child_compassion/models/project_reservation.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Maxime Beck
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+
+from openerp import models, fields
+
+
+class ProjectReservation(models.Model):
+
+    _name = 'compassion.reservation'
+    _description = 'Project Reservation'
+
+    beneficiary_global_id = None
```

Comment:
I didn't meant to declare a field for global_id because according to the [1]specification, this field must always have a value of NULL for an ICP type of reservation.
--
[1] https://github.com/CompassionTCPT/R4-JSON/blob/master/Endpoints/ReadyForTest/GlobalPartners-ICPCreateUpdateReservation_POST.json#L8